src/index.js

```diff
@@ -0,0 +1,16 @@
+import returnInfo from './getInfo';
```

Comment:
## Add weather change appearance feature
- [ ] The project specifications expect a user to see changes in the weather results in its appearance based on the received data. Do you mind adding this? ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
![source](https://user-images.githubusercontent.com/22345961/115111389-d30ac880-9f88-11eb-8fbb-2fd5707a455e.png)